plugins/quay/src/components/QuayDashboardPage/QuayDashboardPage.tsx

```diff
@@ -1,15 +1,9 @@
-import { useEntity } from '@backstage/plugin-catalog-react';
 import React from 'react';
 import { QuayRepository } from '../QuayRepository';
-import { useQuayAppData } from '../useQuayAppData';
+import { useRepository } from '../../hooks';
 
 export const QuayDashboardPage = () => {
-  const { entity } = useEntity();
-  const { repositorySlug } = useQuayAppData({ entity });
-  const info = repositorySlug.split('/');
-
-  const organization = info.shift() as 'string';
-  const repository = info.join('/');
+  const { repository, organization } = useRepository();
```

Comment:
Right now you're using a context to get these values and then simply passing them to `QuayRepository` as props.

I think this would be better if you'd get those values from the context in `QuayRepository` directly. WDYT? Is there any other use for `QuayRepository` when it would not consume the context values anyways and require props?